app.js

```diff
@@ -0,0 +1,22 @@
+// app.js
+// set up ======================================================================
+// get all the tools we need
+var config = require('./config');
+var morgan = require('morgan');
+var path = require('path');
+var express = require('express');
+var app = express.createServer();
+app.listen(config.port);
```

Comment:
the listening should be done on bin/Server -- that way app.js can be required for test and scripts